aut-apps.sh

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env bash
```

Comment:
Let's make this script better by instucting bash to fail on uninitialized vars, any command fail and pipe fails...

```suggestion
#! /usr/bin/env bash

set -euo pipefail

```

Reply:
@RootenberG you still need to accept this suggestion ^